code/modules/networks/computer3/bank_rec.dm

```diff
@@ -0,0 +1,1316 @@
+					else
+						var/leadingZeroCount = length("[src.known_printers.len]")
+						for (var/kp_index=1, kp_index <= src.known_printers.len, kp_index++)
+							dat += "<br> \[[add_zero("[kp_index]",leadingZeroCount)]] [src.known_printers[kp_index]]"
+
+					src.master.temp = null
+					src.print_text("[dat]<br> (0) Return")
+					src.menu = MENU_SELECT_PRINTER
+					return
+				if(MENU_PRINT_OPT_PRINTLOG)
+					src.print_log()
+					return
+
+
+		if(MENU_SELECT_PRINTER)
+			var/printerNumber = round(text2num_safe(command))
+			if (printerNumber == 0)
+				src.menu = MENU_PRINT_SETTINGS
+				src.master.temp = null
+				src.print_settings()
+				return
+
+			if (printerNumber < 1 || printerNumber > src.known_printers.len)
+				return
+
+			src.selected_printer = src.known_printers[printerNumber]
+			src.menu = MENU_PRINT_SETTINGS
+			src.master.temp = null
+			src.print_text("Printer set.")
+			src.print_settings()
+			return
+
+/datum/computer/file/terminal_program/bank_records/proc/mainmenu_text()
+	var/dat = {"<center>B A N K B O S S 2</center><br>
+	Welcome to BankBoss 2<br>
+	<b>Commands:</b>
+	<br>([MENU_MAIN_OPT_INDEX]) View bank records.
+	<br>([MENU_MAIN_OPT_SEARCH]) Search for a record.
+	<br>([MENU_MAIN_OPT_BUDGET]) View station budget.
+	<br>([MENU_MAIN_OPT_PRINT]) Print options.
+	<br>([MENU_MAIN_OPT_QUIT]) Quit."}
+
+	return dat
+
+/datum/computer/file/terminal_program/bank_records/proc/print_budget()
+	src.master.temp = null
+	var/total_funds = global.wagesystem.station_budget + global.wagesystem.research_budget + global.wagesystem.shipping_budget
+	src.print_text("<br><b>Station Budget</b>")
+	src.print_text("Payroll Budget: [num2text(round(global.wagesystem.station_budget),50)][CREDIT_SIGN]")
+	src.print_text("Shipping Budget: [num2text(round(global.wagesystem.shipping_budget),50)][CREDIT_SIGN]")
+	src.print_text("Research Budget: [num2text(round(global.wagesystem.research_budget),50)][CREDIT_SIGN]")
+
+	var/payroll = 0
+	for(var/datum/db_record/R as anything in data_core.bank.records)
+		payroll += R["wage"]
+	var/surplus = round(global.wagesystem.payroll_stipend - payroll)
+	src.print_text("<br><b>Payroll Cycle Details</b>")
+	src.print_text("Per-Cycle Stipend: [num2text(round(global.wagesystem.payroll_stipend), 50)][CREDIT_SIGN]")
+	src.print_text("Per-Cycle Cost: -[num2text(round(payroll),50)][CREDIT_SIGN]")
+	if(surplus >= 0)
+		src.print_text("<b>Payroll Surplus:</b> +[num2text(round(surplus),50)][CREDIT_SIGN]")
+	else
+		src.print_text("<b>Payroll Deficit:</b> [num2text(round(surplus),50)][CREDIT_SIGN]")
+
+	var/net_profit = total_funds - global.wagesystem.total_stipend
+	src.print_text("<br><b>Station Profitability</b>")
+	src.print_text("Total Budget: +[num2text(round(total_funds),50)][CREDIT_SIGN]")
+	src.print_text("Total Stipend: -[num2text(round(global.wagesystem.total_stipend), 50)][CREDIT_SIGN]")
+	src.print_text("<b>Net Profit:</b> [num2text(round(net_profit), 50)][CREDIT_SIGN]")
```

Comment:
```suggestion
```
I still don't think this is useful information to know during a shift and removing it makes the overview soo much clearer to read at a glance (and fit on one default size console window)